### PR TITLE
#57 bug unit tests

### PR DESCRIPTION
**Pull request by deivit98**
Tests now compile , but there is more logic which need to be tested

### CODE REVIEW

**Comment by deivit98** [comment]
Fixed the constants for Article Validation

**Review by MBocharskyi — APPROVED** [review]

File: SportsHub/UnitTests/Controllers/ArticleControllerTests.cs [hunk 1]
```diff
@@ -95,7 +101,8 @@ public async Task GetAllAsync_ArticlesDontExist_ReturnsBadRequest()
         public async Task CreateArticleAsync_ArticleAlreadyExists_ReturnsBadRequest()
         {
             //Arrange
-            var article = ArticleMockData.CreateArticle();
+           // var article = ArticleMockData.CreateArticle();
+            var article = _fixture.Build<CreateArticleDTO>().Create();
```
[the remark this answers is not in the document]
Comment (deivit98) [reply SportsHub/UnitTests/Controllers/ArticleControllerTests.cs:105]: Okey i thought its better approach to build it first . I will have in mind this for now on , thanks

**Review by Banovvv — APPROVED** [review]

File: SportsHub/UnitTests/Controllers/ArticleControllerTests.cs [hunk 2]
```diff
@@ -138,7 +147,8 @@ public async Task EditArticleAsync_NonexistentArticle_ReturnsBadRequest()
         public async Task EditArticleAsync_ValidInput_ReturnsOkStatus()
         {
             //Arrange
-            var article = ArticleMockData.CreateArticle();
+           // var article = ArticleMockData.CreateArticle();
```
Comment (NataliiaBidiuk) [inline SportsHub/UnitTests/Controllers/ArticleControllerTests.cs:150]: remove comment

File: SportsHub/UnitTests/Controllers/ArticleControllerTests.cs [hunk 3]
```diff
@@ -110,7 +117,8 @@ public async Task CreateArticleAsync_ArticleAlreadyExists_ReturnsBadRequest()
         public async Task CreateArticleAsync_NewValidArticle_ReturnsOkStatus()
         {
             //Arrange
-            var article = ArticleMockData.CreateArticle();
+          //  var article = ArticleMockData.CreateArticle();
```
Comment (NataliiaBidiuk) [inline SportsHub/UnitTests/Controllers/ArticleControllerTests.cs:120]: remove comments everywhere

**Review by NataliiaBidiuk — APPROVED** [review]

File: SportsHub/UnitTests/Controllers/ArticleControllerTests.cs (hunk 3 above)
Comment (deivit98) [reply SportsHub/UnitTests/Controllers/ArticleControllerTests.cs]: removed

**Pull request merged**